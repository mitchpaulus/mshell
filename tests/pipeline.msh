[
    [printf "hello\\nEarth!\\n"]
    [sed "s/Earth/World/"]
    [grep World]
] |;

# FILE:pipeline.out
[
    [printf "hello\\nEarth!\\n"]
    [sed "s/Earth/World/"] `pipeline.out` >
] | ;

[
    [printf "hello\\nEarth!\\n"]
    [sed "s/Earth/World/"]
] | os ; wl

# Swapping order of | and os shouldn't matter
[
    [printf "hello\\nEarth!\\n"]
    [sed "s/Earth/World/"]
]  os | ; wl
