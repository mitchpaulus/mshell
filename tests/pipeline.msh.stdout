World!
hello
World!
hello
World!
